services/wallet/api.go

```diff
@@ -524,6 +525,37 @@ func (api *API) GetAddressDetails(ctx context.Context, chainID uint64, address s
 	return result, nil
 }
 
+func (api *API) SignMessage(ctx context.Context, message types.HexBytes, address common.Address, password string) (string, error) {
+	log.Debug("[WalletAPI::SignMessage] message: ", message, " address: ", address, " password: ", password)
```

Comment:
> log password is not a good idea

@qfrank it's ok for the debug, isn't it?

> we don't need such prefix [WalletAPI::SignMessage] anymore, because we have https://github.com/status-im/status-go/pull/4277 in log now

Just followed the approach used in the rest of the file.

Reply:
not good even for debug level IMO

Reply:
except we disable log level `debug` for production build

Reply:
@qfrank I removed the pass from the log. Now I have to add it manually when I'm debugging. ;)